tests/artifacts/config_manager/questions2.csv

```diff
@@ -0,0 +1,20 @@
+question,section,key,default
```

Comment:
do we need questions2?

Reply:
nvm

Reply:
Technically no, but I wanted modularity in the tests so that we could test whether secret sections don't work period or whether only multiple secret sections work.

questions has only secret section, questions2 has two